content/doc/developer/security/misc.adoc

```diff
@@ -134,3 +134,8 @@ Many Jenkins plugins allow the use of Groovy to extend their capabilities.
 For example, the plugin:email-ext[Email Extension Plugin] allows executing a script to determine whether an email should be sent.
 
 All such functionality, if available to users without Overall/Administer, must integrate with plugin:script-security[Script Security Plugin].
+
+=== Logging secrets
+
+When there is a need to log secrets, the logging level should be chosen wisely.
+To prevent accidental exposure in Jenkins' default log recorders, the logging level should be set to a level more verbose than INFO (i.e., CONFIG, FINE, FINER, FINEST).
```

Comment:
"should" is an optional recommendation. While it varies a bit (sometimes there are exceptions to the rule), most of the rest of the page is worded more strongly.

```suggestion
To prevent accidental exposure in Jenkins' default log recorders, the logging level must be set to a level more verbose than INFO (i.e., CONFIG, FINE, FINER, or FINEST).
```

The problem is mostly the [default log](https://www.jenkins.io/doc/book/system-administration/viewing-logs/#logs-on-the-system) (usually stdout)

I'm not sure whether we call the default log a "log recorder", or whether that term is limited to custom ones. I think it's limited to the latter, making this sentence confusing given the UI.